.github/workflows/build-test-windows.yml

```diff
@@ -167,15 +176,22 @@ jobs:
         run: |
           ./scripts/mrbind/msys2_install_clang_ver.sh $(cat scripts/mrbind/clang_version_msys2.txt)
 
-      - name: Build
+      - name: Build (msbuild)
```

Comment:
Can we avoid step renaming and make conditional switch inside? Step name is used currently in Grafana's reports.